files/default/psql-env-ckan.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ "$CKAN_PROPERTY" = "" ]; then
+  CKAN_PROPERTY="sqlalchemy[.]url"
```

Comment:
what about datastore user?

Reply:
Just need to provide the right property to look up.